src/Utilities/functions.jl

```diff
@@ -1697,7 +1669,7 @@ function Base.:/(
         MOI.VectorQuadraticFunction{T},
     },
     g::T,
-) where {T}
+) where {T<:Number}
```

Comment:
Ditto here. The method below assumes `Number`, so these are really inconsistent and a bug fix.